### PR TITLE
[PowerShellV2, BashV3, CmdLineV2] Removed stderr messages truncating

### PR DESCRIPTION
**Pull request by sdobrodeev**
**Task name**: PowerShellV2, BashV3, CmdLineV2

**Description**: Removed stderr messages truncating, because this breaks the masking of secrets in some cases (when the secret is more than 1000 bytes long)

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) Y

**Attached related issue:** (Y/N) 

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/PowerShellV2/task.loc.json [hunk 1]
```diff
@@ -97,6 +97,15 @@
...
       "groupName": "advanced"
     },
+    {
+      "name": "showWarnings",
+      "type": "boolean",
+      "label": "ms-resource:loc.input.label.showWarnings",
+      "required": false,
+      "defaultValue": "false",
+      "helpMarkDown": "ms-resource:loc.input.help.showWarnings",
+      "groupName": "advanced"
```
Comment (egor-bryzgalov) [inline Tasks/PowerShellV2/task.loc.json:107]: Is it redundant change? It has no related input declared in task.json.

**Comment by sdobrodeev** [comment]
@damccorm Don't you have any concerns about removing the stderr truncate logic?

**Review by DaniilShmelev — APPROVED** [review]
LGTM, this limitation seems redundant seeing how other tasks (e.g. [Maven](https://dev.azure.com/v-dshmelev/playground/_build/results?buildId=123&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=ce18b30f-11b8-5f1e-8fbd-391708956df7)) are not at all shy to redirect large amounts of data from standard output

**Review by egor-bryzgalov — APPROVED** [review]

**Review by anatolybolshakov — APPROVED** [review]

**Comment by damccorm** [comment]
> @damccorm Don't you have any concerns about removing the stderr truncate logic?

Sorry, never responded here. I think this is fine. In theory this could get really big/ugly, but it should be ok - masking the secrets is definitely the most important thing here.

**Review by damccorm — APPROVED** [review]

**Pull request merged**